blakserv/ccode.c

```diff
@@ -2344,3 +2351,76 @@ blak_int C_MinigameStringToNumber(int object_id,local_var_type *local_vars,
 	
 	return ret_val.int_val;
 }
+
+// --- BEGIN MULTI-PIPE WEBHOOK SUPPORT ---
+#ifdef BLAK_PLATFORM_WINDOWS

```

Comment:
Move all of the webhook code to a new source file.  It doesn't belong in ccode.c.